### PR TITLE
Fix cypress tests; create Order List Page tests

### PR DESCRIPTION
**Pull request by brandones**
This gets Cypress working and creates a couple working tests for the order list page. Tests for the result entry page are still needed (they nominally exist, but they're broken and need re-writing).

This should also serve as an example to help people create Cypress tests for Order Entry.

### CODE REVIEW

File: app/js/components/LabOrdersList.jsx [hunk 1]
```diff
@@ -444,7 +444,7 @@ export class LabOrdersList extends PureComponent {
       headerClassName: `lab-order-list-column-header lab-order-list-header-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
     }));
     return (
-      <div className="lab-order-list">
+      <div className="lab-order-list" data-cy="order-list">
```
Comment (mogoodrich) [inline app/js/components/LabOrdersList.jsx:447]: interesting, is this a cypress-test specific tag?

Comment (brandones) [reply app/js/components/LabOrdersList.jsx:447]: Yeah, see [Best Practices](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements).

**Comment by brandones** [comment]
> was there a problem that Bamboo wasn't running the Cypress tests, and do we want to enable them if we have them "fixed"?

Yeah definitely! But they're really only half-fixed -- the ones for the entry page are still totally broken. And then yes, we should enable Cypress tests on Bamboo.

**Comment by mogoodrich** [comment]
Cool, go ahead and enable on Bamboo... lmk if you need anything...

**Pull request merged**